plugin/figure/src/main/java/io/deephaven/figure/FigureWidgetTranslator.java

```diff
@@ -518,6 +520,119 @@ private FigureDescriptor.ChartDescriptor translate(ChartImpl chart) {
                                     clientSeries.setPointShape(stringMapWithDefault(mergeShapes(
                                             multiCatSeries.pointShapeSeriesNameToStringMap(),
                                             multiCatSeries.pointShapeSeriesNameToShapeMap())));
+                                } else if (partitionedTableMultiSeries instanceof MultiXYErrorBarSeries) {
+                                    MultiXYErrorBarSeries multiXYErrorBarSeries =
+                                            (MultiXYErrorBarSeries) partitionedTableMultiSeries;
+
+                                    clientAxes.add(makePartitionedTableSourceDescriptor(
+                                            plotHandle, multiXYErrorBarSeries.getX(), SourceType.X, xAxis));
+                                    if (multiXYErrorBarSeries.getDrawXError()) {
+                                        clientAxes.add(makePartitionedTableSourceDescriptor(
+                                                plotHandle, multiXYErrorBarSeries.getXLow(), SourceType.X_LOW, xAxis));
+                                        clientAxes.add(makePartitionedTableSourceDescriptor(
+                                                plotHandle, multiXYErrorBarSeries.getXHigh(), SourceType.X_HIGH,
+                                                xAxis));
+                                    }
+
+                                    clientAxes.add(makePartitionedTableSourceDescriptor(
+                                            plotHandle, multiXYErrorBarSeries.getY(), SourceType.Y, yAxis));
+                                    if (multiXYErrorBarSeries.getDrawYError()) {
+                                        clientAxes.add(makePartitionedTableSourceDescriptor(
+                                                plotHandle, multiXYErrorBarSeries.getYLow(), SourceType.Y_LOW, yAxis));
+                                        clientAxes.add(makePartitionedTableSourceDescriptor(
+                                                plotHandle, multiXYErrorBarSeries.getYHigh(), SourceType.Y_HIGH,
+                                                yAxis));
+                                    }
+
+                                    clientSeries.setLineColor(stringMapWithDefault(mergeColors(
+                                            multiXYErrorBarSeries.lineColorSeriesNameTointMap(),
+                                            multiXYErrorBarSeries.lineColorSeriesNameToStringMap(),
+                                            multiXYErrorBarSeries.lineColorSeriesNameToPaintMap())));
```

Comment:
A lot of these methods seem to be shared by all the `AbstractPartitionedTableHandleMultiSeries` types, and could probably be cleaned up further. I haven't done that, as I didn't want to change too much in this change, and I'm not entirely sure how the autogenerated parts of these series works.